test/e2e/scenario/logs.scenario.ts

```diff
@@ -74,22 +74,15 @@ describe('logs', () => {
       })
     })
 
-  createTest('read only the first bytes of the response')
+  createTest('keep only the first bytes of the response')
     .withLogs({ forwardErrorsToLogs: true })
     .run(async ({ serverEvents, baseUrl, servers }) => {
       await browserExecuteAsync((done) => {
-        fetch('/throw-large-response').then(
-          (response) => {
-            // The body stream needs to be cancelled, else the browser will still download the whole
-            // response even if it is unused.
-            response
-              .body!.getReader()
-              .cancel()
-              .catch((error) => console.log(error))
+        fetch('/throw-large-response')
+          .then((response: Response) => response.clone().blob())
```

Comment:
<!-- Warning represent user-facing problems. If possible, it's great to follow this kind of comment with a suggestion.  -->
**🔨 warning:** ‏the `.clone()` here looks fishy. The user shouldn't need to clone their response... let's see why we need it